src/Fabulous.XamarinForms/Views/Controls/ProgressBar.fs

```diff
@@ -0,0 +1,34 @@
+namespace Fabulous.XamarinForms.Views
+
+open Fabulous.XamarinForms
+open Xamarin.Forms
+open System.Runtime.CompilerServices
+open Fabulous
+
+type IProgressBar =
+    inherit IView
+
+module ProgressBar =
+
+    let WidgetKey = Widgets.register<ProgressBar> ()
+
+    let ProgressColor =
+        Attributes.defineAppThemeBindable<Color> ProgressBar.ProgressColorProperty
+
+    let Progress =
+        Attributes.defineBindable<float> ProgressBar.ProgressProperty
+
+[<AutoOpen>]
+module ProgressBarBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline ProgressBar<'msg>(progress: float) =
+            WidgetBuilder<'msg, ICheckBox>(ProgressBar.WidgetKey, ProgressBar.Progress.WithValue(progress))
```

Comment:
Small mistake: the marker should be `IProgressBar`

Reply:
Opps!! . 😁 Copy and pasta error . Fixed now